app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    file_in_name, file_out_name = command.lstrip("cp").split()
```

Comment:
```suggestion
    command_type, file_in_name, file_out_name = command.split()
```
more clear way is to use unpacking